src/DoctrineAuditBundle/Resources/views/Audit/entry.html.twig

```diff
@@ -57,6 +58,7 @@
                                     {% endif %}
                                 {% endif %}
                             </td>
+                            <td><a href="{{ path('dh_revert', { id: entry.id, object: entry.object_id, field: key, entity: helper.namespaceToParam(entity)}) }}">Revert</a></td>
```

Comment:
other routes are all prefixed with `dh_doctrine_audit_`. This one should follow the same naming scheme.

Reply:
This is an error

Reply:
OK